lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py

```diff
@@ -57,6 +57,7 @@ async def async_execute_single_agent_trade(
     # To do this, we need to manually set the nonce, so we get the base transaction count here
     # and pass in an incrementing nonce per call
     # TODO figure out which exception here to retry on
+    # TODO: change magic number to a constant
```

Comment:
just go ahead and do this with a constant

Reply:
done in ed98988263693013651cc3a203e876610047177d